### ISSUE TITLE
Add all jam games to collection

### ISSUE THREAD

**Issue by fasterthanlime** [itchio/itch.io#907]
Opened by @Alloyed at https://github.com/itchio/itch/issues/1137

**Comment by DaChickenGamer** [comment]
Hello! Is this still on the radar?

**Comment by leafo** [comment]
@DaChickenGamer possibly, we're a little wary about features that can bulk add a ton of stuff with one click since they can be abused. What's your usecase? Big jam, small jam? Into same collection?

**Comment by DaChickenGamer** [comment]
> @DaChickenGamer possibly, we're a little wary about features that can bulk add a ton of stuff with one click since they can be abused. What's your usecase? Big jam, small jam? Into same collection?

This more so relates to the original issud of just have the app being useful for jams like rating and having a button to quickly access them.

**Comment by leafo** [comment]
@DaChickenGamer do you feel like having a better way to browse jam games in the app would be a better solution then, instead of copy to collection? or are you interested in collection functionality specifically

**Comment by DaChickenGamer** [comment]
> @DaChickenGamer do you feel like having a better way to browse jam games in the app would be a better solution then, instead of copy to collection? or are you interested in collection functionality specifically

Yeah I think have a better way to explore jam games or jams in general would be nice.
